scripts/west_commands/runners/openocd.py

```diff
@@ -25,6 +25,7 @@ class OpenOcdBinaryRunner(ZephyrBinaryRunner):
     def __init__(self, cfg, pre_init=None, pre_load=None,
                  load_cmd=None, verify_cmd=None, post_verify=None,
                  tui=None, config=None, serial=None, use_elf=None,
+                 halt_arg=None,
```

Comment:
Please use `no_halt=False` since this is not an argument. It's fine to store `self.halt_arg` as-is, but this kwarg is actually a bool.